Extensions/activity_plus.js

```diff
@@ -468,7 +469,7 @@ XKit.extensions.activity_plus = new Object({
 
 			$(this).addClass("xkit-activity-plus-timestamps-done");
 
-			var m_timestamp = $(this).attr('data-timestamp');
+			var m_timestamp = parseInt($(this).attr('data-timestamp').toString());
```

Comment:
`toString()` shouldn't be necessary for `attr()` since it _always_ returns a string